chain/network/src/test_utils.rs

```diff
@@ -82,8 +82,8 @@ impl PeerInfo {
 #[allow(unreachable_code)]
 pub fn wait_or_panic(max_wait_ms: u64) {
     actix::spawn(delay_for(Duration::from_millis(max_wait_ms)).then(|_| {
-        System::current().stop();
         panic!("Timeout exceeded.");
+        System::current().stop();
```

Comment:
Why do you need to do this? I think if you do this the system will not finish gracefully?

If you keep this order, you probably don't need to do stop anymore, since is not getting there anyway. Even the return statement can be removed AFAIR since panic returns a special value (!) which match everything.

Reply:
@mfornet Is it correct to just remove `System::current().stop();`?

Reply:
If it works this way, we can keep it this way

Reply:
Removed `System::current().stop()`